ports/raspberrypi/supervisor/port.c

```diff
@@ -143,7 +143,9 @@ safe_mode_t port_init(void) {
     never_reset_pin_number(24);
     never_reset_pin_number(25);
     never_reset_pin_number(29);
-    if (cyw43_arch_init()) {
+    // Change this as a placeholder as to how to init with country code.
+    // Default country code is CYW43_COUNTRY_WORLDWIDE)
+    if (cyw43_arch_init_with_country(PICO_CYW43_ARCH_DEFAULT_COUNTRY_CODE)) {
```

Comment:
I filed https://github.com/adafruit/circuitpython/issues/7285 about this as I could not find an existing issue